README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
+kind: AccountClaim
+metadata:
+  creationTimestamp: 2019-07-16T13:52:02Z
+  generation: 2
+  labels:
+    api.openshift.com/id: 00000000000000000000
+    api.openshift.com/name: example-link
+  name: example-link
+  namespace: {NameSpace cluster is being built in}
+  resourceVersion: "54324077"
+  selfLink: /apis/aws.managed.openshift.io/v1alpha1/namespaces/uhc-staging-16toh05i9h8ook5d7ej4al7ejec17rug/accountclaims/razevedo-test2
+  uid: b7113d4a-a8cb-11e9-a2a3-2a2ae2dbcce4
+spec:
+  accountLink: osd-{accountName} (From AccountClaim)
+  aws:
+    regions:
+    - name: us-east-1
+  awsCredentialSecret:
+    name: aws
+    namespace: {NameSpace cluster is being built in}
+  legalEntity:
+    id: 00000000000000
+    name: {Legal Entity Name}
+status:
+  conditions:
+  - lastProbeTime: 2019-07-16T13:52:02Z
+    lastTransitionTime: 2019-07-16T13:52:02Z
+    message: Attempting to claim account
+    reason: AccountClaimed
+    status: "True"
+    type: Unclaimed
+  - lastProbeTime: 2019-07-16T13:52:03Z
+    lastTransitionTime: 2019-07-16T13:52:03Z
+    message: Account claimed by osd-creds-mgmt-fhq2d2
+    reason: AccountClaimed
+    status: "True"
+    type: Claimed
+  state: Ready
+```
+# 3. The controllers
+
+## 3.1. AccountPool Controller
+
+The accountpool-controller is triggered by an accountpool CR or an account CR. It is responsible for filling the Acccount Pool by generating new account CRs. 
+
+It looks at the accountpool CR *spec.poolSize* and it ensures that the number of unclaimed accounts matchs the number of the poolsize. If the number of unclaimed accounts is less then the poolsize it creates a new account CR for the account-controller to process.
+
+### 3.1.1. Constants and Globals
+```
+emailID = "osd-creds-mgmt"
+```
+
+### 3.1.2. Status
+
+Updates accountPool cr 
+```
+  claimedAccounts: 4
+  poolSize: 3
+  unclaimedAccounts: 3
+```
+
+*claimedAccounts* are any accounts with the `status.Claimed=true`
+*unclaimedAccounts* are any accounts with `status.Claimed=false` and `status.State!="Failed"`
+*poolSize* is the poolsize from the accountPool spec
+
+### 3.1.3. Metrics
+
+Updated in the accountPool-controller
+
+```
+MetricTotalAccountCRs
+MetricTotalAccountCRsUnclaimed
+MetricTotalAccountCRsClaimed
+MetricTotalAccountPendingVerification
+MetricTotalAccountCRsFailed
+MetricTotalAccountCRsReady
+MetricTotalAccountClaimCRs
+
+```
+
+## 3.2. Account Controller
+
+The account-controller is triggered by an account CR. It is responsible for following behaviours:
+
+If the *awsLimit* set in the constants is not exceeded
+1. Creates a new account in the organization belonging to credentials in secret `aws-account-operator-credentials` 
+2. Configure Users from *iamUserNameUHC* and *iamUserNameSRE*
+    * Creates IAM user in new account 
+    * Attaches Admin policy
+    * Generates a secret access key for the user 
+    * Stores user secret in a AWS secret
+3. Creates STS CLI Credentials for SRE
```

Comment:
Creates STS CLI tokens
Creates Federated webconsole URL using the *iamUserNameSRE* user